inc/command/class-command.php

```diff
@@ -663,6 +750,69 @@ protected function stop_browser_container( InputInterface $input, OutputInterfac
 		return $return_var;
 	}
 
+	/**
+	 * Return suite files within a tests folder.
+	 *
+	 * @param string $folder Tests folder to scan.
+	 * @return array
+	 */
+	protected function get_test_suites( $folder ) : array {
+		$suites = glob( $folder . '/*.suite.yml' );
+		foreach ( $suites as $i => $suite ) {
+			$suites[ $i ] = substr( $suite, strlen( $folder ) + 1, strpos( $suite, '.suite.yml' ) - strlen( $suite ) );
+		}
+
+		return $suites;
+	}
+
+	/**
+	 * Returns whether a suite configuration requires a specific module.
+	 *
+	 * @param string $suite_file Suite YML file.
+	 * @param string $module Module name.
+	 *
+	 * @return boolean
+	 */
+	protected function suite_has_module( string $suite_file, string $module ) : bool {
+		$suite_config = file_get_contents( $suite_file );
+		$needs_web_driver = preg_match( "#- {$module}#", $suite_config );
+
+		return $needs_web_driver;
+	}
+
+	/**
+	 * Return the test suite selected from CLI arguments.
+	 *
+	 * @param InputInterface $input
+	 *
+	 * @return string
+	 */
+	protected function get_test_suite_argument( InputInterface $input ) : string {
+		$arguments = $input->getArguments();
+		$first_argument = $arguments['options'][1] ?? '';
+
+		// Test suite should be the first argumet so we can check for it, otherwise, assume no suite is selected.
+		if ( 0 === strpos( $first_argument, '-' ) ) {
```

Comment:
Not sure I follow why there'd definitely be a dash here. Shouldn't this method check if we've called `codecept run` too?

What I ran `composer dev-tools codecept g:cest acceptance SomeName` for example.

Reply:
The goal here was to exclude any parameter argument, eg: `-p`, `--path`, etc.

I've done some more tests with a few variations, all seems to be returning the suite name without even checking if it's a parameter or option argument.

> composer dev-tools codecept g:cest acceptance SomeName
composer dev-tools codecept run acceptance
composer dev-tools codecept run -p some/path acceptance
composer dev-tools codecept -p some/path run acceptance

However, it'll fail if the suite name is passed after the end of options signal `--`, which I think is an edge case.

So I'll remove that condition.

Reply:
Ah, now I remember, it was these formats that might have failed, because the arguments after the end of options signal aren't parsed, unlike anything before it:

eg: this will fail:
> composer dev-tools codecept run -- -p test acceptance

this will work:
> composer dev-tools codecept run -p test acceptance

I'll try to find a way to mitigate that.

Reply:
I don't think we need to support `composer dev-tools codecept run -- -p test acceptance`, doesn't seem like it is a standard approach anyways.